cmd/common/st/remove.go

```diff
@@ -8,21 +8,21 @@ import (
 	"github.com/wal-g/wal-g/pkg/storages/storage"
 )
 
-const deleteObjectShortDescription = "Delete the specified storage object"
+const removeShortDescription = "Removes objects by the prefix from the specified storage"
 
-// deleteObjectCmd represents the deleteObject command
-var deleteObjectCmd = &cobra.Command{
-	Use:   "rm relative_object_path",
-	Short: deleteObjectShortDescription,
+// removeCmd represents the deleteObject command
+var removeCmd = &cobra.Command{
```

Comment:
When we support failover storages in `backup-fetch` command, we can get rid of the most part of copy-paste in `pg_full_backup_failover_storage_test` and `pg_transfer_backup_test` Docker tests.
However, in these tests, we also need a way for clearing all the storages. Before, the `delete` command was used for that in tests, and that command doesn't support failover storages. So I just supported removing multiple files by a prefix in this `st rm` command, because it already supports failover storages and it required really a few changes.